### DISCUSSION TITLE
Where is the changes log for ComfyUI?

### DISCUSSION THREAD

**Discussion by andreszs** [Comfy-Org/ComfyUI_frontend#2099] [Q&A] [answered]
So, a new ComfyUI frontend version has been published today, nice. I wonder if anyone can answer these questions:

1. Where is the **changelog** file to see exactly what's new in this version?
2. According to **package.json**, this is version 1.6.11. I know this is a question for the Manager developer, however, why does the manger say I have _ComfyUI v0.3.10-14-ga90aafa (2024-12-30)_ instead?

**Comment by christian-byrne** [comment, marked as answer]
> Where is the changelog file to see exactly what's new in this version?

[Releases](https://github.com/Comfy-Org/ComfyUI_frontend/releases)

> According to package.json, this is version 1.6.11. I know this is a question for the Manager developer, however, why does the manger say I have ComfyUI v0.3.10-14-ga90aafa (2024-12-30) instead?

Separate versioning:

- ComfyUI version 0.3.10
- ComfyUI_frontend version 1.6.11
